demisto_sdk/commands/common/hook_validations/content_entity_with_test_playbooks_validator.py

```diff
@@ -0,0 +1,42 @@
+from abc import ABC
+
+from demisto_sdk.commands.common.hook_validations.content_entity_validator import \
+    ContentEntityValidator
+from demisto_sdk.commands.common.hook_validations.structure import \
+    StructureValidator
+from demisto_sdk.commands.common.hook_validations.test_playbook import \
+    TestPlaybookValidator
+
+
+class ContentEntityWithTestPlaybooksValidator(ContentEntityValidator, ABC):
```

Comment:
I have an idea - let's put this in the id_set_validator (`demisto_sdk/commands/common/hook_validations/id.py`) This is more akin to a "third-party" validator for every file type and since this validation does require the existence of id_set file we should add it there.

Reply:
what do you think about putting this to the ConfJsonValidator instead? The check and the conf.json are much more related and I could add the relevant logic inside validate_manager so it will run when needed